saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -482,25 +489,27 @@ msgid ""
 "If Saas is to be used for all of CDS, select FC 110804. If the Saas is to be "
 "used specifically for your BU, please select the appropriate FC."
 msgstr ""
-"Si ce SaaS doit être utilisé par tous le personnel du SNC, sélectionnez FC 110804. Si le"
-"Saas doit être utiliséutilisé spécifiquement pour votre BU, veuillez "
-"sélectionner le FC approprié."
+"Si ce SaaS doit être utilisé par tous le personnel du SNC, sélectionnez FC "
+"110804. Si leSaas doit être utiliséutilisé spécifiquement pour votre BU, "
```

Comment:
There should be a space between `le` and `SaaS`
And we should remove one of the two `utilisé`